R/install_spark_versions.R

```diff
@@ -1,149 +0,0 @@
-.globals <- new.env(parent = emptyenv())
```

Comment:
Looks like we already did that but forgot to remove this instance, so this code was effectively duplicated, see: https://github.com/rstudio/sparklyr/blob/c436791ea967e2ad0fbdab36009479bf864fe0af/R/spark_globals.R